src/rules/rename-default-import.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @fileoverview Rule to enforce aliases for default imports
+ * @author Michał Kołodziejski
+ */
+
+import docsUrl from '../docsUrl'
+
+
+function isDefaultImport(specifier) {
+  if (specifier.type === 'ImportDefaultSpecifier') {
+    return true
+  } else if (specifier.type === 'ImportSpecifier' && specifier.imported.name === 'default') {
```

Comment:
There’s no need for the else here; this function can be two if’s and one return false

Reply:
there shouldn't be an "else" after a return. This should just be two if's, and one return false.